src/backend/executor/index_scan_executor.cpp

```diff
@@ -130,6 +130,22 @@ bool IndexScanExecutor::DExecute() {
 bool IndexScanExecutor::ExecIndexLookup() {
   assert(!done_);
 
+  /*
+   * If query is IN+Subquery, the values should be set using context params
+   * for now, the flag params_exec_ is set to 1 only in nestloop join
+   * in this case, the params is the results of the outer plan
+   * We can add more cases in future
+   */
+  //TODO: we probably need to add more for other cases in the future
```

Comment:
I think we should chat about this during the group meeting. I think that we can do this stuff in the transformation module and leave the executors unchanged.